kbase-extension/static/kbase/js/widgets/function_output/kbaseReadsViewer.js

```diff
@@ -302,6 +391,13 @@ define (
 		}
 */
 
+        get_uuid4: function() {
```

Comment:
There's a utility module that has a UUID function in it, so you don't need to make one in every widget.
Here's how to use it:
https://github.com/kbase/narrative/blob/develop/docs/developer/widget_helper_modules.md#utilstring